rollup.config.js

```diff
@@ -44,6 +45,13 @@ function withPlugins(
     }
     basePlugins.push(babel(options.babelConfig));
   }
+  if (!options.enableAssertions) {
+    basePlugins.push(
+      strip({
+        functions: ['assert', 'assertNotReached', 'assertExists', 'ES.assertExists']
```

Comment:
Do we need to do anything to remove the emission of the source code for these functions from prod builds? Or is this automatic via this plugin?

Reply:
We pass `enableAssertions: !isProduction` later in the file. So it's not automatic, but already taken care of.